From 6c566e5bcda49c1adfe1ba3f2276789f95890d7c Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Tue, 20 Feb 2024 17:44:56 +0100
Subject: [PATCH] arm64: defconfig: activate lptimer as tick broadcast

During the low power modes the generic ARM timer is deactivated, so the
the tick broadcast need to be used in low power mode,
base on LPTIMER3 which is clocked by LSE on STMicroelectronics boards.

Change-Id: Id4d683e5603a44903ad94e9fda92dd7895df0636
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/361427
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -1,4 +1,5 @@
 #  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-02-defconfig-addons.config -o fragment-02-defconfig-addons.config
+CONFIG_CLKSRC_STM32_LP=y
 CONFIG_COUNTER=m
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_IIO_STM32_LPTIMER_TRIGGER=m
